packages/website/docusaurus.config.js

```diff
@@ -32,6 +32,7 @@ const config = {
         customCss: require.resolve('./src/css/custom.css'),
       },
     ],
+    '@docusaurus/theme-search-algolia',
```

Comment:
Just a kind notice: we may introduce shorthands in the next (few) release (facebook/docusaurus#5930) like ESLint's `eslint-plugin-react` -> `plugins: ['react']`. In that case you may want to put this in `themes` instead so you can shorten it to `themes: ['search-algolia']`